jfrog-applications/jfrog-cli/cli-command-summaries.md

```diff
@@ -0,0 +1,117 @@
+# CLI Command Summaries
+
+## Overview
+
+The command summaries feature allows the recording of JFrog CLI command outputs into the local file system. This can be used to display a summary of the executed CLI command.
+
+Each command execution that has implemented this feature can save data files into the file system. These files will then be used to create a summary markdown.
+
+## How to use?
+
+To use the command summaries, you'll need to set the `JFROG_CLI_COMMAND_SUMMARY_OUTPUT_DIR` environment variable.
+
+After that, supported commands will start saving data files to this directory, along with markdown files.
+
+### ⚠️ Notice: Delete your files after multiple executions
+
+The CLI doesn't delete the files itself as they need to persist after more than one execution, so note that it is your responsibility to clear after you in your pipelines.
+
+## How to implement? 👩‍💻
```

Comment:
Let's put all sections that are related to the development of Summaries umder a "Notes for developers" section.